app/Models/Task.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    use HasFactory;
+
+    protected $guarded = [];
+
+    protected $casts = [
+        'closed_at' => 'datetime',
+    ];
+
+    public function recipient()
```

Comment:
suggestion: re: names et al
```suggestion
    /**
     * The user for which the task is assigned.
     */
    public function assignee()
```